dephell_pythons/_finder.py

```diff
@@ -100,7 +108,10 @@ def get_version(self, path: Path) -> str:
 
         # get version from interpreter for other implementations (like pypy)
         command = r'print(__import__("sys").version)'
-        result = subprocess.run([str(path), '-c', command], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        try:
+            result = subprocess.run([str(path), '-c', command], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
What is the text of the exception you've caught here as well?

Reply:
In this place I added try/except just in case.

Reply:
Let's leave it as it was before for a while. Here we are executing the given python with known params. An error can mean only one from two:

1. Our python lookup was broken, we should fix it earlier.
1. Users passed the wrong python path inside, let him know.

In both cases, we should not suppress the error